packages/mgt-chat/src/components/ChatListItem/ChatListItem.tsx

```diff
@@ -0,0 +1,170 @@
+  },
+  profileImage: {
+    width: '60px', // Increase the width for a bigger image
+    height: '60px', // Increase the height for a bigger image
+    ...shorthands.borderRadius('50%'), // This will make it round
+    marginRight: '10px',
+    objectFit: 'cover', // This ensures the image covers the area without stretching
+    position: 'relative', // Ensures the ::before pseudo-element is positioned relative to the profileImg
+    display: 'flex',
+    ...shorthands.overflow('hidden'), // Ensures no part of the ::before spills out
+    alignItems: 'center', // This will vertically center the image
+    justifyContent: 'center' // This will horizontally center the image
+  },
+  chatInfo: {
+    alignSelf: 'left',
+    alignItems: 'center',
+    ...shorthands.padding('5px')
+  },
+  chatTitle: {
+    textAlign: 'left',
+    ...shorthands.margin('0'),
+    fontSize: '1em',
+    color: '#333',
+    textOverflow: 'ellipsis',
+    ...shorthands.overflow('hidden'),
+    whiteSpace: 'nowrap',
+    width: '300px'
+  },
+  chatMessage: {
+    textAlign: 'left',
+    ...shorthands.margin('0'),
+    fontSize: '0.9em',
+    color: '#666',
+    textOverflow: 'ellipsis',
+    ...shorthands.overflow('hidden'),
+    whiteSpace: 'nowrap',
+    width: '300px'
+  },
+  chatTimestamp: {
+    textAlign: 'right',
+    alignSelf: 'start',
+    marginLeft: 'auto',
+    paddingLeft: '10px',
+    fontSize: '0.8em',
+    color: '#999',
+    width: '60px'
+  }
+});
+
+export const ChatListItem = ({ chat, myId, onSelected }: IMgtChatListItemProps & IChatListItemInteractionProps) => {
+  const styles = useStyles();
+
+  // shortcut if no valid user
+  if (!myId) {
+    return <></>;
+  }
+
+  // Copied and modified from the sample ChatItem.tsx
+  // Determines the title in the case of 1:1 and self chats
+  const inferTitle = (chatObj: Chat) => {
+    if (myId && chatObj.chatType === 'oneOnOne' && chatObj.members) {
+      const other = chatObj.members.find(m => (m as AadUserConversationMember).userId !== myId);
+      const me = chatObj.members.find(m => (m as AadUserConversationMember).userId === myId);
+      return other
+        ? `${other?.displayName || (other as AadUserConversationMember)?.email || other?.id}`
+        : `${me} (You)`;
+    }
+    return chatObj.topic || chatObj.chatType;
+  };
+
+  const extractTimestamp = (timestamp: NullableOption<string> | undefined): string => {
+    if (timestamp === undefined || timestamp === null) return '';
+    const currentDate = new Date();
+    const date = new Date(timestamp);
+
+    const [month, day, year] = [date.getMonth(), date.getDate(), date.getFullYear()];
+    const [currentMonth, currentDay, currentYear] = [
+      currentDate.getMonth(),
+      currentDate.getDate(),
+      currentDate.getFullYear()
+    ];
+
+    if (currentDay === day && currentMonth === month && currentYear === year) {
+      return date.toLocaleTimeString([], { hour: 'numeric', minute: '2-digit' });
```

Comment:
I'm wondering if there would be possible to use something like `navigator.language` to support some formatting preferences the user has. As an example, if the language is "fr", the time would be shown with a 24-hour clock, and the short date would be shown as day/month. I did a quick test in Teams and saw that it does follow that convention when the language is switched.